shared/chat/conversation/input-area/normal/shared.tsx

```diff
@@ -19,7 +20,41 @@ export const ExplodingMeta = ({explodingModeSeconds}: {explodingModeSeconds: num
   )
 }
 
+type BotCommandUpdateStatusProps = {
+  status: RPCChatTypes.UIBotCommandsUpdateStatus
+}
+
+export class BotCommandUpdateStatus extends React.Component<BotCommandUpdateStatusProps> {
```

Comment:
should just be a function component, we're trying to reduce classes we don't need

Reply:
Is there anything wrong with classes? I just liked having these things in the same namespace, and having props floating around on this.

Reply:
there's overhead we can remove. it can all be teh same just not a class. i don't understand what you mean by 'same namespace' and having 'props floating around'